### PR TITLE
feat: created home page

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
Screenshot (64)
Screenshot (65)

- Added Home page 
- added all the images in assets.
- added cards and background image.

### CODE REVIEW

**Review by shivangisharma01 — COMMENTED** [review]
looks great and easily readable code.

**Review by shivangisharma01 — APPROVED** [review]

File: home.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+}
+
+.flex-row{
+    align-items: center;
+    gap: 0;
+}
+
+.btn-link-noBorder{
+    color: #fffffe;
```
Comment (nakulsharma15) [inline home.css]: Just a suggestion, you can use variables for colors

**Review by nakulsharma15 — APPROVED** [review]
Gave a small suggestion. Everything else seems good

File: home.css (hunk 1 above)
Comment (Tanjum786) [inline home.css]: use variables for the colors.

**Review by Tanjum786 — APPROVED** [review]
Everything looks good. use variables for colors and before merging do beautifly or prettier for proper indent

File: home.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply home.css]: fixed it in the latest commit

Comment (vaibhav-mahalle) [reply home.css]: fixed it int the latest commit

**Pull request merged**